ipad/Models/Shift/Form Fields/ShiftFormHelper.swift

```diff
@@ -120,4 +132,9 @@ class ShiftFormHelper {
         return columns
     }
     
+    private static func stationRequired(_ station: String?) -> Bool {
```

Comment:
function to check if one of our 3 required stations that I have reused in another file (may be better to import later)

Reply:
Early return 👍